src/UI.js

```diff
@@ -0,0 +1,162 @@
+    this.commentAuthors = [];
+    this.searchInput = document.querySelector('#search-input');
+    this.titleInput = document.querySelector('title-input');
+    this.bodyInput = document.querySelector('.body-input');
+    this.commentSubmit = document.querySelector('#comment-submit-btn');
+  }
+
+  // Show movies
+  async showMovies(movies) {
+    const likeArray = await getLikes();
+    let output = '';
+    let likeCount = 0;
+
+    // let commentCount = 0;
+    // console.log('edw like array', likeArray);
+    // let isMovieId = 0;
+
+    movies.forEach((movie) => {
+      likeArray.forEach((like) => {
+        // const idString = parseInt(movie.show.id, 10);
+        // console.log(like.likes, like.item_id);
+        // eslint-disable-next-line eqeqeq
+        if (like.item_id == movie.show.id) {
+          // console.log(like.likes);
+          likeCount = like.likes;
+          // console.log(likeShow);
+          // return likeCount;
+        }
+        // like.item_id = like.item_id.toString();
+        // console.log(like.likes);
+      });
+
+      const image = movie.show.image?.medium ?? 'https://i.ibb.co/nPzyFm6/placeholder.png';
+      output += `
+      <div class="card mb-5 border border-5 shadows" id="${movie.show.id}" style="max-width: 20rem, max-height: 20rem;">
+        <img src="${image}" class="card-img-top" alt="">
+        <div class="card-body">
+          <h5 class="card-title">${movie.show.name}</h5>
+          <p class="card-text">Premiered: ${movie.show.premiered}</p>
+          <button class="like-btn btn btn-dark">  
+            <i type="button" class="far fa-thumbs-up">${likeCount}</i>
+          </button>
+          
+          <p class="card-text"></p>
+          <div class="d-flex justify-content-between">
+            
+            <button type="button" class="btn btn-primary comments-btn">
+              Comments
+            </button>
+
+            <div class="ml-3 btn btn-primary reservations-btn">Reservations</div>
+          </div>
+        </div>
+      </div>
+      `;
+    });
+
+    this.movie.innerHTML = output;
+
+    const movieKids = this.movie.children;
+
+    const openPop = document.getElementById('overlay');
+    const closePop = document.getElementsByClassName('close-pop');
+    const movieName = document.getElementsByClassName('movie-name');
+    const movieImage = document.getElementsByClassName('movie-image');
+    const movieYear = document.getElementsByClassName('movie-year');
+    const movieDesc = document.getElementsByClassName('description');
+    const commentAuthor = document.querySelector('#author-input');
+    const commentBody = document.querySelector('#comment-input');
+
+    function openCommentsModal(sourceId) {
+      console.log('works');
+      closePop.addEventListener('click', () => {
+        if (!openPop.classList.contains('hidden')) {
+          console.log('ok');
+          openPop.addClass(' hidden');
+        }
+      });
```

Comment:
- [ ] I should see details about the selected item
- [ ] I should retrieve data from the API to show the item comment
- [ ] After filling the form and clicking the submit button, nothing happens, I also couldn't close the modal as well, please fix this issues

Reply:
those are in progress.

Reply:
this is still not fix yet

Reply:
- [ ] This issue is still not fixed yet.